src/main/resources/templates/components/delivery.ftl

```diff
@@ -69,9 +69,8 @@
     </ul>
     [/#if]
 
-
     [#assign review = delivery.getReview()!]
-    [#if review?? && review?has_content && (review.getGrade()?? || review.getCommentary()??)]
+    [#if review?? && seeGrade && review?has_content && (review.getGrade()?? || review.getCommentary()??)]
```

Comment:
Yes, not ideal, but we will change that when we start transitioning to a more MVC alike pattern, and not in this PR 😄